.github/scripts/badwords.pl

```diff
@@ -56,13 +56,13 @@ sub file {
         foreach my $w (@w) {
             my $case = $exactcase{$w};
             if(($in =~ /^(.*)$w/i && !$case) ||
-               ($in =~ /^(.*)$w/ && $case) ) {
+                ($in =~ /^(.*)$w/ && $case) ) {
```

Comment:
I argue this is wrong. It was already correct, now it made it bad. The first character here should be aligned with the second parenthesis on the line above.